File: python/raindrops/raindrops.py
def convert(number: int) -> str:
    result = ''
    if not number % 3:
        result += 'Pling'
    if not number % 5:
        result += 'Plang'
    if not number % 7:
        result += 'Plong'

    return result or str(number)

File: go/raindrops/raindrops.go
package raindrops
import "strconv"

// Convert converts a number into a string that contains raindrop sounds
func Convert(inp int) string {
	res := ""
	if inp%3 == 0 {
		res += "Pling"
	}
	if inp%5 == 0 {
		res += "Plang"
	}
	if inp%7 == 0 {
		res += "Plong"
	}
	if res == "" {
		res = strconv.Itoa(inp)
	}
	return res
}